meshio/gmsh/common.py

```diff
@@ -152,9 +153,12 @@ def _gmsh_to_meshio_order(cells):
         # fmt: off
         "tetra10": [0, 1, 2, 3, 4, 5, 6, 7, 9, 8],
         "hexahedron20": [
-            0, 1, 2, 3, 4, 5, 6, 7, 8, 11, 16,
-            9, 17, 10, 18, 19, 12, 15, 13, 14
-        ],
+            0, 1, 2, 3, 4, 5, 6, 7, 8, 11, 13,
+            9, 16, 18, 19, 17, 10, 12, 14, 15,
+        ],  # https://vtk.org/doc/release/4.2/html/classvtkQuadraticHexahedron.html
```

Comment:
These are links to VTK, but you're manipulating the Gmsh format. Perhaps you're editing the wrong file?

Reply:
as I understood, meshio follows VTK conventions for cell numbering, or am I wrong? I got this numbering from comparison with the corresponding cell in here: http://www.manpagez.com/info/gmsh/gmsh-2.2.6/gmsh_65.php#SEC65

So what I am doing is to give the re ordering from gmsh (gmsh format, see link above) and meshio/vtk (see link in PR)

Reply:
That's a nice link! Can you add it to the code too? (Otherwise I'll forget about it.)